provider/lib/home/extra_actions_button.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 The Flutter Architecture Sample Authors. All rights reserved.
+// Use of this source code is governed by the MIT license that can be found
+// in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:todos_app_core/todos_app_core.dart';
+
+import '../todo_list_model.dart';
+
+class ExtraActionsButton extends StatelessWidget {
+  const ExtraActionsButton({
+    Key key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final store = Provider.of<TodoListModel>(context);
+
+    return PopupMenuButton<ExtraAction>(
+      key: ArchSampleKeys.extraActionsButton,
+      onSelected: (action) {
```

Comment:
I wish this function was directly on `PopupMenuItem` instead. It weird to read this way 🤔